junifer/configs/juseless/datagrabbers/ucla.py

```diff
@@ -0,0 +1,120 @@
+"""Provide a concrete implementation for UCLA dataset."""
+
+# Authors: Federico Raimondo <f.raimondo@fz-juelich.de>
+#          Leonard Sasse <l.sasse@fz-juelich.de>
+# License: AGPL
+
+from pathlib import Path
+from typing import List, Union
+
+from ....datagrabber import PatternDataGrabber
+from ....api.decorators import register_datagrabber
+from ....utils import raise_error
+
+
+@register_datagrabber
+class JuselessUCLA(PatternDataGrabber):
+    """Concrete implementation for pattern-based data fetching of UCLA data.
+
+    Parameters
+    ----------
+    datadir : str or Path, optional
+        The directory where the dataset is stored
+        (default '/data/project/psychosis_thalamus/data/fmriprep').
+    **kwargs
+        Keyword arguments passed to superclass.
+
+    """
+
+    def __init__(
+        self,
+        datadir: Union[
+            str, Path, None
+        ] = "/data/project/psychosis_thalamus/data/fmriprep",
+        tasks: Union[str, List[str], None] = None,
+        **kwargs,
```

Comment:
It's not being used by the superclass so can be safely removed.

Reply:
which argument? the tasks and **kwargs?

Reply:
just kwargs, right? makes sense

Reply:
Yeah.